### PR TITLE
Added Portuguese translation

### PR DESCRIPTION
**Pull request by maiconkkl**

### CODE REVIEW

File: locales/br.lua [hunk 1]
```diff
@@ -0,0 +1,5 @@
+Locales ['br'] = {
+  ['message_title'] = '^3Veículo Fechado',
+  ['message_locked']  = 'fechado',
+  ['message_unlocked'] = 'aberto',
+}
```
Comment (HawaiiBeach) [inline locales/br.lua:5]: trailing space missing

**Review by ElPumpo — CHANGES_REQUESTED** [review]

File: locales/br.lua [hunk 2]
```diff
@@ -1,5 +1,6 @@
 Locales ['br'] = {
-  ['message_title'] = '^3Veículo Fechado',
-  ['message_locked']  = 'fechado',
+  ['message_title']    = '^3Veículo Fechado',
```
Comment (HawaiiBeach) [inline locales/br.lua]: why add extra spaces?

**Review by ElPumpo — CHANGES_REQUESTED** [review]

File: locales/br.lua (hunk 2 above)
Comment (maiconkkl) [reply locales/br.lua]: For better visualization the variables

Comment (HawaiiBeach) [reply locales/br.lua]: no

Comment (maiconkkl) [reply locales/br.lua]: Fixed

**Comment by HawaiiBeach** [comment]
ty

**Pull request merged**